pkg/services/sqlstore/migrations/ualert/tables.go

```diff
@@ -323,4 +323,7 @@ func AddAlertAdminConfigMigrations(mg *migrator.Migrator) {
 
 	mg.AddMigration("create_ngalert_configuration_table", migrator.NewAddTableMigration(adminConfiguration))
 	mg.AddMigration("add index in ngalert_configuration on org_id column", migrator.NewAddIndexMigration(adminConfiguration, adminConfiguration.Indices[0]))
+	mg.AddMigration("add column send_alerts_to in ngalert_configuration", migrator.NewAddColumnMigration(adminConfiguration, &migrator.Column{
+		Name: "send_alerts_to", Type: migrator.DB_NVarchar, Length: 190, Nullable: false, Default: "'all'",
```

Comment:
I'm just wondering how we ended up with `190`? I'm not suggesting we should change it, but it sounds quite specific so wondering if we chose it for a reason?

Reply:
You're right. I saw that number being used for most `VARCHAR` field types and thought it was a default, but maybe it would be better to use the length of the longest string in the enum as the length of the field.

One potential issue we could have in the future is needing to add another option. In that case we would either need to avoid exceeding the character limit, or to write a migration to change the schema. TBH I don't think we'd run into a problem like that since we're currently covering all possible cases (`internal`, `external`, `all`).

What do you think?

Reply:
Personally, I use integer. I think string value in database does not make sense:
1. No questions about chosen length. Nobody likes magic numbers in code. 
2. The database is not intended to be user-friendly. 
3. In the future when we may add a new value to the enum we will have to make sure their string representation will fit in the column and\or add migration to expand. 
4. Size. It's really not important on this scale but if we start nitpicking it's 4 bytes (for integer, and even less if we play game with smallint, byte) vs 8 bytes (+2 additional for varchar in mysql). Again, at this scale, this is a negligible difference.